lib/network/modules/components/nodes/util/CircleImageBase.js

```diff
@@ -47,7 +47,7 @@ class CircleImageBase extends NodeBase {
    * Set the images for this node.
    *
    * The images can be updated after the initial setting of options;
-   * therefore, this method needs to be reentrant. 
+   * therefore, this method needs to be reentrant.
```

Comment:
Please, don't reformat files, especially don't if you don't change anything in it.

Reply:
Visual Studio code removing trailing spaces...

Reply:
I don't care what tool you use to remove them, just don't. The problem is that you'll be credited as the author of these lines and when I need to find out why was something introduced your name and commit will pop up simply because you didn't turn off reformatting in VS Code.